src/org/usfirst/frc4904/robot/autonomous/strategies/AutonGearCenterPegVision.java

```diff
@@ -0,0 +1,33 @@
+package org.usfirst.frc4904.robot.autonomous.strategies;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.GearioOuttake;
+import org.usfirst.frc4904.sovereignty.strategies.GearAlign;
+import org.usfirst.frc4904.standard.commands.RunAllSequential;
+import org.usfirst.frc4904.standard.commands.RunFor;
+import org.usfirst.frc4904.standard.commands.chassis.ChassisConstant;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+
+public class AutonGearCenterPegVision extends CommandGroup {
+	public static final double TIME_INITIAL_APPROACH = 1.25;
+	public static final double OUTTAKE_TIME_TOTAL = 3;
```

Comment:
I do not think this is the best name for this variable, unless I am misunderstanding its use.